File: python/1061.py
dia, idia = input().split()
ih, p, im, p, ise = input().split()
dia, fdia = input().split()
fh, p, fm, p, fse = input().split()

fdia = int(fdia)
idia = int(idia)

totali = ((int(ih) * 3600) + (int(im) * 60) + int(ise))
totalf = ((int(fh) * 3600) + (int(fm) * 60) + int(fse))

total = totalf - totali

if (total > 0):
    ih = total / 3600
    im = (total % 3600) / 60
    ise = (total % 3600) % 60
    idia = fdia - idia
elif (total < 0):
    total = 86400 + total
    ih = total / 3600
    im = (total % 3600) / 60
    ise = (total % 3600) % 60
    idia = fdia - idia - 1
else:
    ih = 0
    im = 0
    ise = 0
    idia = fdia - idia
    
print("%d dia(s)\n%d hora(s)\n%d minuto(s)\n%d segundo(s)\n" % (idia, ih, im, ise))

File: 1061.c
#include <stdio.h>

int main(){
    int ih, im, is, idia, fh, fm, fs, fdia, totalseg;
    char aux[4];
    scanf("%s %d",aux, &idia);
    scanf("%d : %d : %d", &ih,&im,&is);
    scanf("%s %d",aux, &fdia);
    scanf("%d : %d : %d", &fh,&fm,&fs);

    totalseg = ((fh * 3600) + (fm * 60) + fs) - ((ih * 3600) + (im * 60) + is);

    if (totalseg > 0){
        ih = totalseg / 3600;
        im = (totalseg % 3600) / 60;
        is = (totalseg % 3600) % 60;
        idia = fdia - idia;
    }else if (totalseg < 0){
        totalseg = 86400 + totalseg;
        ih = totalseg / 3600;
        im = (totalseg % 3600) / 60;
        is = (totalseg % 3600) % 60;
        idia = fdia - idia - 1;
    } else {
        ih = 0;
        im = 0;
        is = 0;
        idia = fdia - idia;
    }
    
    printf("%d dia(s)\n%d hora(s)\n%d minuto(s)\n%d segundo(s)\n",idia, ih, im, is);
    return 0;
}